lab_2_keywords_cooccurrence/main.py

```diff
@@ -40,7 +63,10 @@ def calculate_frequencies_for_content_words(candidate_keyword_phrases: KeyPhrase
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not type_check(candidate_keyword_phrases, list):
+        return None
+    return {token: sum(look.count(token) for look in candidate_keyword_phrases)
```

Comment:
3 `for`s in one line? excess. Try to write simple and understandable code

Reply:
Fixed